bin/test

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env bash
+YARP_FIXTURES_DIR=test/fixtures/yarp/test/fixtures
```

Comment:
Sorry this is `test/yarp/fixtures` now

Reply:
Thank you for raising this change 🙏 I did the change.
For some mysterious reason, two files trigger an error:

```
Assertion failed: (list->size == 0 || newline_offset > list->offsets[list->size - 1]), function yp_newline_list_append, file yp_newline_list.c, line 36.
rake aborted!
SignalException: SIGABRT
```

Files: 
- https://github.com/ruby/yarp/blob/main/test/yarp/fixtures/spanning_heredoc.txt
- https://github.com/ruby/yarp/blob/main/test/yarp/fixtures/newline_terminated.txt

I'll have to dig in this issue, but for now I can't understand why [this line](https://github.com/ruby/yarp/blob/main/src/util/yp_newline_list.c#L36) from YARP is run when we run tests.

If someone has an idea, do not hesitate 👍

Reply:
Indeed, it's been corrected. I'm now waiting for a fix in Rubocop before pushing, I've already opened an issue. At this point, I wonder if it wouldn't be better to wait for the YARP migration, to remove the 6 `SyntaxTree` failures at the same time. WDYT?

Anyway, I'm keeping monitoring this PR, it's not abandoned in the wild. 😄